frontend/src/components/core/Container.tsx

```diff
@@ -10,6 +10,27 @@ const createContainer = (Container: any, props: MyProps) => {
   return <Container>{children}</Container>;
```

Comment:
Unrelated to your PR but I just realized that this code I wrote is probably completely unnecessary haha